### ISSUE TITLE
qfit_protein crashes on writing multiconformer_model

### ISSUE THREAD

**Issue by blake-riley** [ExcitedStates/qfit-3.0#65]
I tried to run qFit on my favourite protein (PDB:2BIJ) last night, but it crashed while writing multiconformer_model.pdb.

Not sure why this is yet. It managed to get out the first 100 or so residues, but then died without writing the remaining 200 of them.

```
Traceback (most recent call last):
  File "/home/briley/.pyenv/versions/miniconda3-4.3.30/envs/qfit3.1.0_dev+parallelrefactor/bin/qfit_protein", line 11, in <module>
    load_entry_point('qfit', 'console_scripts', 'qfit_protein')()
  File "/home/briley/qfit-3.1/src/qfit/qfit_protein.py", line 485, in main
    multiconformer = qfit.run()
  File "/home/briley/qfit-3.1/src/qfit/qfit_protein.py", line 194, in run
    multiconformer = self._run_qfit_residue_parallel()
  File "/home/briley/qfit-3.1/src/qfit/qfit_protein.py", line 299, in _run_qfit_residue_parallel
    multiconformer.tofile(fname)
  File "/home/briley/qfit-3.1/src/qfit/structure/base_structure.py", line 192, in tofile
    PDBFile.write(fname, self)
  File "/home/briley/qfit-3.1/src/qfit/structure/pdbfile.py", line 99, in write
    f.write(CoorRecord.line1.format(*field))
TypeError: unsupported format string passed to NoneType.__format__
```

Will be looking into CoorRecord.line1 and why it's NoneType, and not a string.

**Comment by blake-riley** [comment]
Digging backwards, the `field` appears as
`['ATOM', 3367, 'OE2', 'A', 'GLU', 'A', 390, '', -2227.82, nan, nan, None, None, '', '']`
x,y,z are ... plainly wrong.

This is true further back in `A_390/multiconformer_residue.pdb`
where we have regions that look like (first atom is well-formatted, remainder are broke)
```
ATOM      7  CD AGLU A 390     -38.365 -10.914 -32.729  0.20 27.21           C
ATOM      8  OE1AGLU A 390    -70493.249-31862.285-49436.635  0.20 27.21           O
ATOM      9  OE2AGLU A 390    -2227.823-1000.848-1569.376  0.20 27.21           O
```

and also broken in `A_390/conformer_1.pdb`:
```
ATOM      7  CD  GLU A 390     -38.365 -10.914 -32.729  0.20 27.21           C
ATOM      8  OE1 GLU A 390    -70493.249-31862.285-49436.635  0.20 27.21           O
ATOM      9  OE2 GLU A 390    -2227.823-1000.848-1569.376  0.20 27.21           O
```

Why?

**Comment by blake-riley** [comment]
This should be fixed as a result of #72 and #82 .
